where-to-travel/src/main/webapp/script.js

```diff
@@ -246,8 +282,12 @@ function submitDataListener(event) {
   if (home == null) {
     const content = '<p> No home location found. Please set a home location and try again.</p>';
     openModal(content);
-  }
-  else {
+  } else if ($('#' + FEEDBACK_ID).children().length >= 1) {
+    const content = '<p> Please enter valid search parameters</p>';
+    openModal(content);
+  } else {
+    console.log($('#' + FEEDBACK_ID).children().length);
```

Comment:
Are these log statements necessary to keep for non-debugging purposes?

Reply:
b46438b fixed